starters/gatsby-starter-minimal/src/pages/index.js

```diff
@@ -27,42 +27,72 @@ const codeStyles = {
 const listStyles = {
   marginBottom: 96,
   paddingLeft: 0,
-  listStyleType: "none",
 }
 const listItemStyles = {
-  marginBottom: 12,
   fontWeight: "300",
   letterSpacing: 1,
+  fontSize: "25px",
```

Comment:
```suggestion
  fontSize: "24px",
```

Does that work for those bullet things still?